### PR TITLE
Adding new page for CSS Grid Animation support

### PR DESCRIPTION
**Pull request by jensimmons**
Attempt No 2. 
As discussed in #6298.
Supported in Firefox since FF66. https://hacks.mozilla.org/2019/03/firefox-66-the-sound-of-silence/
Supported in Safari 16 and STP. https://developer.apple.com/documentation/safari-release-notes/safari-16-release-notes
Not supported in Chromium browsers (I tested using CodePen demo at https://css-irl.info/animating-css-grid/)

### CODE REVIEW

**Comment by jensimmons** [comment]
I'm not sure what's wrong. I don't understand the error message. Probably something to do with the very end of the file. But everything regarding the content about Grids + animation, and about browser support is correct. 

Can someone who knows the Can I Use build process better fix this file?

**Comment by Schweinepriester** [comment]
I'm not totally sure, but maybe it's the spacing, as in it maybe wants 2 spaces per level? Currently it seems to be tabs...

File: features-json/css-grid-animation.json [hunk 1]
```diff
@@ -0,0 +1,491 @@
+{
+	"title":"CSS Grid animation",
+	"description":"As defined in CSS Grid level 1, grid-template-columns and grid-template-rows are properties that can be animated.",
+	"spec":"https://www.w3.org/TR/css-grid-1/",
+	"status":"cr",
+	"links":[
+		{
```
Comment (Fyrd) [inline features-json/css-grid-animation.json:7]: I think it's the empty "link" that it's got a problem with, just need to change this to an empty array. :)

Comment (jensimmons) [reply features-json/css-grid-animation.json]: Thanks. Fixed. Let's see if that passes.

**Comment by jensimmons** [comment]
Grr.

**Pull request closed without merging**